src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Diagnostics/CSharpDiagnosticsEndToEndTest.cs

```diff
@@ -43,11 +43,23 @@ public void M()
         await ValidateDiagnosticsAsync(input);
     }
 
-    private async Task ValidateDiagnosticsAsync(string input)
+    [Fact]
+    public async Task Handle_Razor()
```

Comment:
I put this here to try to test my changes, then realized that these tests don't actually send LSP messages to our server, only Roslyn, but figured I'd leave it anyway.